react/Header/UserAccountMenu/UserAccountMenu.less

```diff
@@ -61,10 +61,11 @@
   opacity: 0.5;
 }
 
-.subItem {
-  @media @desktop {
-    background-color: @sk-gray-light;
-  }
+.menuSeparator {
+  height: 1px;
+  border: 0;
+  background-color: @sk-mid-gray-light;
+  margin: 5px 8px;
```

Comment:
This throws it off grid. These should be a multiple of the grid row, minus 1 to account for element the height.

Reply:
Changed these to comply with the grid